sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -263,22 +250,20 @@ internal class RadarApiClient(
             val usingRemoteTrackingOptions = RadarSettings.getTracking(context) && RadarSettings.getRemoteTrackingOptions(context) != null
             params.putOpt("usingRemoteTrackingOptions", usingRemoteTrackingOptions)
             params.putOpt("locationServicesProvider", RadarSettings.getLocationServicesProvider(context))
-            params.putOpt("rooted", RadarUtils.isRooted())
+            if (verified) {
+                params.putOpt("integrityToken", integrityToken)
+                params.putOpt("integrityException", integrityException)
+            }
         } catch (e: JSONException) {
             callback?.onComplete(RadarStatus.ERROR_BAD_REQUEST)
 
             return
         }
 
-        val host = RadarSettings.getHost(context)
-        val uri = Uri.parse(host).buildUpon()
-            .appendEncodedPath("v1/track")
-            .build()
-        val url = URL(uri.toString())
-
+        val path = "v1/track"
         val headers = headers(publishableKey)
 
-        apiHelper.request(context, "POST", url, headers, params, true, object : RadarApiHelper.RadarApiCallback {
+        apiHelper.request(context, "POST", path, headers, params, true, object : RadarApiHelper.RadarApiCallback {
```

Comment:
@nickpatrick - it looks like `verified` should be passed through to this call

Reply:
Good catch, fixed